schedule/yast/modify_existing_partition/modify_existing_partition_x86_64.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: modify_existing_partition
+description: >
+  Installation where we modify some pre-existing partitions. Must depend on some
+  create_hdd test suite.
+vars:
+  YUI_REST_API: 1
+schedule:
+  expert_partitioning:
```

Comment:
I don't see "expert_partitioning" in the default, is it normal?

Reply:
expert_partitioning is in the part1 PR, but I added it here

Reply:
Ah, ok, I did not know about the other PR.